src/containers/Header/index.jsx

```diff
@@ -21,15 +21,17 @@ const Header = (props) => {
         >
           <Img src={enterpriseLogo || EdxLogo} alt="" />
         </Link>
-        <Dropdown
-          title="edx@example.com"
-          menuItems={[
-            {
-              label: 'Logout',
-              href: '/',
-            },
-          ]}
-        />
+        {props.location.pathname.indexOf('/login') === -1 &&
```

Comment:
This might look cleaner by using `mapStateToProps` to pass in `state.login.isAuthenticated` rather than using the routers location.